docs/source/API/core/policies/ThreadVectorMDRange.rst

```diff
@@ -1,52 +1,60 @@
-# `ThreadVectorMDRange`
+``ThreadVectorMDRange``
+=======================
 
-Header File: `Kokkos_Core.hpp`
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
 
-Usage: 
-  ```c++
     parallel_for(ThreadVectorMDRange<Kokkos::Rank<...>, TeamHandle>(team, extent1, extent2, ...),
       [=] (int i1, int i2, ...) {...});
-    parallel_reduce(ThreadVectorMDRange<Kokkos::Rank<...>, TeamHandle>(team, extent1, extent2, ...), 
+    parallel_reduce(ThreadVectorMDRange<Kokkos::Rank<...>, TeamHandle>(team, extent1, extent2, ...),
       [=] (int i1, int i2, ..., double& lsum) {...}, sum);
-  ```
 
-ThreadVectorMDRange is a [nested execution policy](https://kokkos.github.io/kokkos-core-wiki/ProgrammingGuide/HierarchicalParallelism.html?highlight=nested#nested-parallelism)
-used inside of hierarchical parallelism. 
 
-## Interface
-  ```c++
+ThreadVectorMDRange is a `nested execution policy <./NestedPolicies.html>`_  used inside of hierarchical parallelism.
+
+Interface
+---------
+
+.. code-block:: cpp
+
    template <unsigned N, ..., typename TeamHandle>
-   struct ThreadVectorMDRange<Rank<N, ...>, TeamHandle> {
+   struct ThreadVectorMDRange<Rank<N, ...>, TeamHandle>
+   {
      ThreadVectorMDRange(team, extent1, extent2, ..., extentN) { /* ... */ }
-
-     /* ... */
    };
-  ```
 
-## Description
```

Comment:
Are lines 33 -34 an overview of what `ThreadVectorMDRange` does?  If so please see comment above about `Usage`.  Following the recently-adopted standard for API entries, `Arguments` would become `Parameters`, and `Requirements` would be expressed as required parameters (vs. optional).